### ISSUE TITLE
[Question] Is there way to run multiple tests in parallel within the same test suite from terminal by passing some flag instead of using .parallel syntax ?

### ISSUE THREAD

**Issue by amrsa1** [microsoft/playwright#10510]

**Comment by mxschmitt** [comment]
If I understand you correctly, that you want parallelism without `describe.parallel` on a per test basis, no there is not as of today.

**Comment by Amrkamel1** [comment]
@mxschmitt yes this is what I meant, replace .parallel with some option or flag that can be used in terminal

**Comment by mxschmitt** [comment]
This is not yet possible. We could probably re-formulate it to a feature request like: Add a mode so that all the tests run in parallel by default?

**Comment by Amrkamel1** [comment]
Yes this will definetly be a big plus

**Comment by dgozman** [comment]
@Amrkamel1 What's your scenario? The more information you provide, the better we can help.

I'd imagine you have a few tests in one file, and you just want to quickly run them all in parallel? How do you run them - passing a file name? Why don't you mark them as `describe.parallel`?

**Comment by Amrkamel1** [comment]
@dgozman  some times we have like hundreds of test cases running them locally is a bit hard from the machine, but we still need to run them in parallel against our servers during CICD, so instead of keeping manually changing the the describe block to describe. parallel was thinking passing a flag in command running in the pipeline would be a better solution

**Comment by dgozman** [comment]
@Amrkamel1 We usually suggest to alter your config for CI vs local run, so that you don't have to twiddle with command line args. For example, mark tests as `describe.parallel` and configure the number of workers differently:

```js
const config = {
  workers: process.env.CI ? 6 : 1,
};
```

Does that work for you?

**Comment by Amrkamel1** [comment]
> @Amrkamel1 We usually suggest to alter your config for CI vs local run, so that you don't have to twiddle with command line args. For example, mark tests as `describe.parallel` and configure the number of workers differently:
> 
> ```js
> const config = {
>   workers: process.env.CI ? 6 : 1,
> };
> ```
> 
> Does that work for you?

Yes im currently im using this way, it will run test suite in parallel but not test cases.

**Comment by rwoll** [comment]
If I understand this request correctly, we'd find this useful to. 

For context, we're moving lots of API tests (ie tests that don't require the browser, but take advantage of @playwright/test as a generalized amazing test framework).

Most of the API tests run as end to end tests against live environments (sort of like smoke tests). Most of the test files are wrapped in a giant test.describe.parallel block.

I'd like to remove those wraps, and set a config in playwright.config.ts to opt into parallel by default for all tests within a file. The few files that then will require non-parallel tests, I'd like to mark with a test.parallel(false) or test.serialize() at the top of the test file.

If this request does not align with the original filing of this ticket, please let me know and I will refile as another issue. 

Thanks!
